pkg/vm/process/types.go

```diff
@@ -212,6 +212,13 @@ func (proc *Process) GetLastInsertID() uint64 {
 	return 0
 }
 
+func (proc *Process) SetCacheForAutoCol(name string) {
```

Comment:
we suddenly add this logic?    what does this mean?    I don't know the implicatoin of this.

there has to be some comment on what is going on